crates/macros/Cargo.toml

```diff
@@ -20,7 +20,6 @@ gadget-clients = { workspace = true, optional = true }
 gadget-event-listeners = { workspace = true, optional = true }
 gadget-logging = { workspace = true, optional = true }
 gadget-crypto = { workspace = true, optional = true }
-gadget-testing-utils = { workspace = true, optional = true }
```

Comment:
Not sure why we're re-exporting testing from macros, I don't think it's a good idea if we want to use macros in testing.